masu/external/__init__.py

```diff
@@ -25,3 +25,24 @@
 
 # Notificaton Definitions
 AWS_SNS_HEADER_MSG_TYPE = 'X-Amz-Sns-Message-Type'
+AWS_SNS_HEADER_TOPIC_ARN = 'X-Amz-Sns-Topic-Arn'
+
+# Valid AWS Regions from https://docs.aws.amazon.com/general/latest/gr/rande.html
+AWS_REGIONS = ['us-east-2',
```

Comment:
There's another copy of the aws regions in tests/external/downloader/aws/__init__.py

Let's pick one to keep and remove the other, and update the references.  The version used for testing obviously needs to be static, but the app itself could also use boto's `EC2.Client.describe_regions()` call to fetch a current list, so that we don't have to maintain this as AWS adds/changes their region list.

Reply:
yeah I thought about using the describe_regions() function but figured we would need to have some sort of default region to start with to use for that client.  I'll consolidate.